models/articles.models.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection')
+
+exports.selectArticleById = (article_id) => {
+    return db.query(`
+        SELECT author, title, article_id, topic, body, votes, article_img_url, created_at::varchar FROM ARTICLES
+        WHERE article_id = $1`, [article_id])
+        .then(({rows}) => {
+            if(rows.length < 1){                
+                return Promise.reject(
+                    {status: 404 , msg: 'Not Found'}
+                )
+            }
+            console.log(rows[0])
```

Comment:
Remember to remove any console.logs before submitting a pull request - you don't want console.logs (especially those used for debugging!) in production code.